components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/base/JsBaseAuthenticationContext.java

```diff
@@ -0,0 +1,106 @@
+package org.wso2.carbon.identity.application.authentication.framework.config.model.graph.js.base;
```

Comment:
License header

Reply:
Added, corrected in other places as well